cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -260,6 +260,21 @@ public void scanApplications(Cursor cursor,
     }
   }
 
+  public Map<ApplicationId, ApplicationMeta> getAllApplications() throws IOException {
```

Comment:
we shouldn't have a method that does a full table scan like this. The existing getAllApplications(namespace) should be used instead.